modules/age.nix

```diff
@@ -5,13 +5,47 @@ with lib;
 let
   cfg = config.age;
 
+  isDarwin = pkgs.stdenv.isDarwin;
```

Comment:
This is technically checking for the wrong thing. You would want to know if the current module evaluator is `nix-darwin` not if the package set passed is for darwin. I know for the most part those two situations overlap, but I think it would be a good idea to try and find something more specific to check for. Perhaps an option that only exists in nix-darwin could be used.

Reply:
> Perhaps an option that only exists in nix-darwin could be used.

There's quite a few of those, but many would be overly-specific for this type of check (e.g. anything setting system defaults). One of the following options should always be specific to nix-darwin but generic enough to look friendly:

- `environment.darwinConfig` [[link](https://github.com/LnL7/nix-darwin/blob/1df878b6f8351795a3bebfbe4fd2d02e1e8b29d6/modules/environment/default.nix#L65-L76)]
- `launchd.*` [[link](https://github.com/LnL7/nix-darwin/blob/1df878b6f8351795a3bebfbe4fd2d02e1e8b29d6/modules/launchd/default.nix#L137-L151)] (or simply `launchd` but I'm not sure that it's an independent option):